internal/service/handlers/play.go

```diff
@@ -475,19 +475,62 @@ func handlePlayRequest(ctx context.Context, s *discordgo.Session, m *discordgo.M
 	urlStr := args["url"]
 
 	pid := p.PlaylistID()
+	playPack := make(chan service.PlayCall, 1)
 
-	if handled, err := processPlaylist(ctx, pid, s, m, p, urlStr); err != nil {
+	p.Enqueue(playPack)
```

Comment:
There is no guarantee that other play requests won't race here to send their package channel. Even a buffered channel won't stop the race condition which still occurs when the buffer is full and even when trying to tell a buffer witj capacity to save to the queue. I wonder if discord has any guarantees here on handler calls and handler concurrency. I am at the mercy of discord-go's handler implementation. This at least guarantees an order if a playlist arg play event is fired.